infra.bs

```diff
@@ -258,11 +262,31 @@ inclusive.
 
 <h3 id=strings>Strings</h3>
 
-<p>A <dfn export>string</dfn> is a sequence of <a>code points</a>. Strings are denoted by double
-quotes and monospace font.
+<p>A <dfn export>JavaScript string</dfn> is a sequence of unsigned 16-bit integers, also known as
+<dfn export lt="code unit">code units</dfn>. A <a>JavaScript string</a> can also be interpreted as
+containing <a>code points</a>, per the conversion defined in <a>The String Type</a> section of the
+JavaScript specification. [[!ECMA-262]]
+
+<p>A <dfn export>scalar value string</dfn> is a sequence of <a>scalar values</a>. These
+<a>scalar values</a> can also be addressed as <a>code points</a>.
+
+<p class=note>A <a>scalar value string</a> is useful for any kind of I/O or other kind of operation
+where <a>UTF-8 encode</a> comes into play.
```

Comment:
I think we need to expand this reasoning a bit more, given how confused people are about when to use USVString in Web IDL.